src/components/Layout/Layout.tsx

```diff
@@ -1,8 +1,51 @@
-import { Outlet } from 'react-router-dom';
+import { useContext } from 'react';
+import { Outlet, Navigate, useLocation } from 'react-router-dom';
+import Step from '@mui/material/Step';
+import Stepper from '@mui/material/Stepper';
+import StepContent from '@mui/material/StepContent';
+import StepLabel from '@mui/material/StepLabel';
+import FormScreens from 'constants/formScreens';
+import routes from 'constants/routes';
+import WizardFormContext from 'contexts/WizardFormContext';
+
+const STEPS = Object.keys(FormScreens);
 
 export default function Layout() {
+  const { formValues } = useContext(WizardFormContext);
+
+  function getActiveStep(path: string) {
+    switch (path) {
+      case routes.PERSONAL_DETAILS:
+        return formValues.PERSONAL_DETAILS.order;
+      case routes.DELIVERY_MODE:
+        return formValues.DELIVERY_MODE.order;
+      case routes.PAYMENT_METHOD:
+        return formValues.PAYMENT_METHOD.order;
+      case routes.FORM_SUBMISSION:
+        return formValues.FORM_SUBMISSION.order;
+      case routes.FORM_SUCCESS:
+        return formValues.FORM_SUCCESS.order;
+      default:
+        return formValues.PERSONAL_DETAILS.order;
```

Comment:
raise error here

Reply:
we can only use numbers for activeStep prop in MUI Stepper